pkg/server/middleware.go

```diff
@@ -25,7 +25,7 @@ func authMiddlewareWithUser(a *auth.Authenticator, handlerFunc func(user *auth.U
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		user, err := a.Authenticate(r)
 		if err != nil {
-			klog.Infof("authentication failed: %v", err)
+			klog.V(4).Infof("authentication failed: %v", err)
```

Comment:
Will this get logged by default? We want to avoid the named cookie not present errors in the log, which is a red herring.

Reply:
So the default log level `Normal` which is represented by `2`, so no it should not be logged by default. Only if the console logLevel is set to `Debug`, which is represented by `4`.

Reply:
You have everything at 4, but some of these we want to see by default. (See my initial review.)

Reply:
Ping @jhadvig, this comment is still pending :)